browser/ui/tabs/brave_tab_style.h

```diff
@@ -77,35 +74,6 @@ class BraveTabStyle : public TabStyleBase {
   }
 
   int GetSeparatorCornerRadius() const override { return 0; }
-
-  SkColor GetTabBackgroundColor(
-      const TabStyleBase::TabSelectionState state,
-      bool hovered,
-      const bool frame_active,
-      const ui::ColorProvider& color_provider) const override {
-    const SkColor active_color = color_provider.GetColor(
-        frame_active ? kColorTabBackgroundActiveFrameActive
-                     : kColorTabBackgroundActiveFrameInactive);
-    const SkColor inactive_color = color_provider.GetColor(
-        frame_active ? kColorTabBackgroundInactiveFrameActive
-                     : kColorTabBackgroundInactiveFrameInactive);
-
-    if (hovered) {
```

Comment:
If we consider hover state, it's same with base class method. So, deleted this overriding.